doc/pairs.md

```diff
@@ -207,3 +207,65 @@ tuples
     - 51
 ...
 ```
+## ``fields`` parameter
+
+With ``fields`` parameter you can choose the value of which fields you would like to get.
+Result contains fields specified by ``fields`` parameter, primary key and scan key. We need primary and scan keys in result to support pagination.
+
+**Example:**
+
+```lua
+objects = {}
+-- condition by indexed non-unique non-primary field
+for _, obj in crud.pairs('developers', {{'>=', 'age', 31}},  {use_tomap = true, fields = {'name'}}) do
+    table.insert(objects, obj)
+end
+
+objects
+---
+- - id: 6
+    name: Alexey
+    age: 31
+  - id: 4
+    name: Mikhail
+    age: 51
+...
+```
+
+You can use pagination and ``fields`` option together if you pass tuple gotten with ``fields`` option to ``after``.
```

Comment:
We need to highlight that `after` tuple should contain the same fields as we receive on `pairs` call with such `fields` parameters.

Smth like

Wrong:

```lua
tuples = {}
for _, tuple in crud.pairs('developers', {{'>=', 'age', 27}} }) do  -- `fields` isn't specified
    table.insert(tuples, tuple)
end

-- THIS WOULD FAIL
-- call `pairs` with `fields` option specified 
-- and pass to `after` tuple that were got without `fields` option
crud.pairs('developers', {{'>=', 'age', 27}}, {fields = {'name'}}, after = tuples[1] })
```